src/main/java/org/thoughtcrime/securesms/connect/DcEventCenter.java

```diff
@@ -187,7 +188,26 @@ public long handleEvent(@NonNull DcEvent event) {
         return 0;
     }
 
+    boolean devMode = Prefs.isDeveloperModeEnabled(context);
+    String logPrefix = devMode? "[accId="+accountId + "] " : "";
+
     if (accountId != context.dcContext.getAccountId()) {
```

Comment:
`devMode` and `logPrefix` are needed only inside the branch and could be final:

```suggestion
    if (accountId != context.dcContext.getAccountId()) {
        final boolean devMode = Prefs.isDeveloperModeEnabled(context);
        final String logPrefix = devMode? "[accId="+accountId + "] " : "";
```